app/Lib/Rscripts/analyse.R

```diff
@@ -55,7 +55,7 @@ Analyse <- function(key, input.answers, number.answeroptions) {
         if (!is.null(input.answers[i, j]) & all(key[, j] == 0)) {
           input.correct[i, j] <- input.answers[i, j]
         } else {
-          if(any(input.answers[i, j] == which(key[, j] == 1))) {
+          if(any(input.answers[i, j] == which(key[, j] == 1))) 
```

Comment:
Please, check the style guide about spacing (https://google.github.io/styleguide/Rguide.xml#spacing) and curly braces (https://google.github.io/styleguide/Rguide.xml#curlybraces).